src/components/RadioGroup/RadioGroup.vue

```diff
@@ -1,20 +1,26 @@
 <template>
   <fieldset
     :class="[
-      'border-none m-0 p-0 flex',
-      {'flex-col' : separateLines}
+      'border-none flex'
     ]"
   >
-    <legend :class="[{'text-sm font-normal normal-case tracking-normal text-gray-500 mb-1 border-b-0':!srOnlyLegend}, {'sr-only': srOnlyLegend}]">
+    <legend :class="[{'type-small-700 text-black mb-1.5':!srOnlyLegend}, {'sr-only': srOnlyLegend}]">
```

Comment:
given they are exclusive:
```suggestion
    <legend :class="srOnlyLegend ? 'sr-only' : 'type-small-700 text-black mb-1.5':!srOnlyLegend'">
```